app/static/Climate-Mind_bundled.yml

```diff
@@ -170,12 +170,28 @@ paths:
               schema:
                 description: ''
                 type: object
+                x-examples:
+                  example-1:
+                    conversationId: cc63e48f-d066-44e7-851c-c28af17ab3fb
+                    userA:
+                      name: Bill
+                      id: ba5df442-7261-4fc1-bff0-5dfd84035d56
+                      sessionId: 671e4949-a3e4-4844-b9d2-cd843f48f357
+                    userB:
+                      name: Bob
+                    conversationStatus: 2
+                    consent: true
```

Comment:
Is this meant to say consent=false? (Instead of true)

Reply:
Hmm, this is under x-examples:example-1, which I don't think I created and I don't see in the interface. Unless I'm missing something, I suggest I add this to my low-priority todos.